### DISCUSSION TITLE
Does adftools publishing support both data factories connected to DevOps Git repos

### DISCUSSION THREAD

**Discussion by mbremer99** [Azure-Player/azure.datafactory.tools#209] [General]
Hello,

we have a setup with two data factories - both of them being connected to their individual Azure Git repo.
I created a deployment process with DevOps pipelines. According to the logs all works fine, but there are no changes visible in the target factory.

Is this setup supported? If yes, how do I make it work?

Rgds,
Markus

**Comment by NowinskiK** [comment]
I think you need to provide more details.

**Comment by mbremer99** [comment]
Hello,

the setup is as follows with regard to data factory:
- DEV environment: ADF DEV with Azure DevOps repo DEV_Repo
- PROD environment: ADF PROD with Azure DevOps report PROD_Repo
Not your typical setup but required for several reasons.

The plan is to use adftools via DevOps pipelines for deployment from DEV to PROD.
Two Pipelines (artifact creation and release pipeline) have been setup and run successfully. All logs show success.
But there are no changes visible in ADF PROD.
For testing I created a new data factory pipeline and changed an existing data factory pipeline in DEV environment. The PROD environment does not show the new pipeline and the existing pipeline still shows the old setup.
So it looks like it is not working.
The only difference compared to a small test setup I did before is that both environments are connected to their own DevOps repo. 

That's why I'm wondering if this setup is supported by adftools?

Best Regards,
Markus

**Reply by NowinskiK** [reply]
You should NOT configure repo for an environment different than DEV. Full stop.
This setup is terribly wrong. And very risky. You should stop doing this unless you're aware of all consequences.

adftools probably deploy artifacts to ADF PROD (LIVE mode), but I guess you are still working with its dev mode when having GIT repo configured on this environment. Double check what you see in PROD/LIVE mode.
